src/_Form.sass

```diff
@@ -25,7 +25,7 @@ select
 	box-shadow: none
 	box-sizing: inherit // Forced to replace inherit values of the normalize.css
 	height: 3.8rem
-	padding: .6rem 1.0rem // The .6rem vertically centers text on FF, ignored by Webkit
+	padding: .6rem 1.0rem .5rem // This vertically centers text on FF, ignored by Webkit
```

Comment:
@ndbroadbent  Maybe I'm wrong. But using Roboto, it's aligned better with `padding: .5rem 1.0rem .7rem`.